src/util/locking/scripts/RedisLuaScripts.ts

```diff
@@ -0,0 +1,151 @@
+import type { Callback, Redis as RedisInterface } from 'ioredis';
+import { InternalServerError } from '../../errors/InternalServerError';
+
+const SUFFIX_WLOCK = '.wlock';
+const SUFFIX_LOCK = '.lock';
+const SUFFIX_COUNT = '.count';
+const LOCKED = 'locked';
+
+/**
+ * Lua scripts to be used as Redis operations.
+ */
+export const SCRIPTS = {
```

Comment:
```suggestion
export const REDIS_LUA_SCRIPTS = {
```
Since this name is exported over the entire project.